charter.md

```diff
@@ -1,52 +1,124 @@
-  Kubernetes project, and what privileges that entails
-* Decide which sub-projects are part of the Kubernetes project, including
-  accepting new sub-projects and pruning existing sub-projects to maintain
-  community focus
+  Kubernetes project, and what privileges that entails.
+* Control and delegate access to and establish processes regarding
+  any Kubernetes repository<sup>[4](#footnote4)</sup> 
+* Control and delegate access to and establish processes regarding
+  project resources/assets<sup>[5](#footnote5)</sup>  
+
+## Delegated responsibilities of note
+
+Following are noteworthy responsibilities that have been delegated and *should
+be present in the relevant charters*.  Additional delegated responsibilities 
+should also be specified in a charter document for the 
+party<sup>[2](#footnote2)</sup>.
+
+### SIG Architecture
+
+* Define, evolve, and defend the technical vision / mission and scope of the
+  project.
+* Decide which SIG is responsible for each technical area of the 
+  project.
+  * Decisions should be codified in SIG charters.
+  * Changes to or creation of SIG charters should still be overseen (approved)
+    by the Steering Committee.
+* Provide oversight pertaining to which sub-projects created by SIGs are part
+  of the project.  If necessary, direct sub-projects outside the scope of the 
+  Kubernetes project to be moved outside the project.
+* Define what a conformant Kubernetes cluster is as part of any certification
+  process.
+  * Facilitated through the conformance Working Group.
+
+### SIG Release
+
 * Decide how and when official releases of Kubernetes artifacts are made and
   what they include
-* Control access to, establish processes regarding, and provide a final
-  escalation path for any Kubernetes repository, which currently includes all
-  repositories under the github organizations kubernetes, kubernetes-incubator,
-  kubernetes-security, kubernetes-client, etc. and is expected to expand in the
-  future
-* Control and delegate access to and establish processes regarding other
-  project resources/assets, including artifact repositories, build and test
-  infrastructure, web sites and their domains, blogs, social-media accounts,
-  etc.
-* Define what a conformant Kubernetes cluster is as part of any certification
-  process
-* Manage the Kubernetes brand to decide which things can be called “Kubernetes”
-  and how that mark can be used in relation to other efforts or vendors.
-* Declare a release, so that the committee can ensure quality/feature/other
-  requirements are met.
-* Request funds and other support from the CNCF (e.g. marketing, press, etc.)
\ No newline at end of file
+* Control access to and establish processes regarding release related project
+  resources/assets, including artifact repositories, build and test
+  infrastructure, etc.
+* Declare a release and ensure quality/feature/other requirements are met.
+
+### Code of Conduct Committee
```

Comment:
same.

Reply:
Some of this content was in the Steering charter originally. It looks like Phil was trying to be clear about to which subgroups those responsibilities were delegated. The alternative is to be vague here (e.g., don't mention code of conduct at all). Maybe we could categorize high-level areas, such as technical areas, policies, processes, etc., and just state that they should be delegated to subgroups, either SIGs or committees.